draft-ietf-quic-transport.md

```diff
@@ -2874,6 +2882,18 @@ sender to become limited by the congestion controller (as described in
 receiver. Therefore, a sender SHOULD ensure that other frames are sent in
 addition to PADDING frames to elicit acknowledgments from the receiver.
 
+An endpoint that is only sending acknowledgements will not receive
+acknowledgments from its peer unless those acknowledgements are included in
```

Comment:
```suggestion
acknowledgments from its peer.
```

Reply:
Isn't the existing text more accurate(minus the sender/endpoint change)?